exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/core/query/lucene/SearchIndex.java

```diff
@@ -1054,6 +1059,50 @@ private void reindex(boolean doReindexing, boolean doForceReindexing, boolean do
       }
    }
 
+   /**
+    * Deletes all the files from the swap directory
+    */
+   private void cleanupSwapDirectory()
+   {
+      PrivilegedAction<Void> action = new PrivilegedAction<Void>()
+      {
+         public Void run()
+         {
+            String WorkspaceName = getContext().getContainer().getWorkspaceName();
+            File directory = SpoolConfig.getSwapPath(WorkspaceName);
+            if(directory != null)
+            {
+               File[] files = directory.listFiles();
+               if (files != null && files.length > 0)
+               {
+                  log.info("Force delete swap files after re-index " + WorkspaceName);
+                  for (int i = 0; i < files.length; i++)
+                  {
+                     File file = files[i];
+                     try
+                     {
+                        SwapFile swap = SwapFile.get(directory, file.getName());
+                        if(swap != null)
+                        {
+                           swap.delete();
+                        }
+                     }
+                     catch (IOException e)
+                     {
+                        if(log.isDebugEnabled())
```

Comment:
This condition should be removed. The log must be displayed whenever something goes wrong and it should be an error, not a warn IMO.